catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/utils/BrickFormulaEditTextDataInteractionWrapper.java

```diff
@@ -102,4 +103,23 @@ public BrickFormulaEditTextDataInteractionWrapper performEnterString(int stringR
 				.perform(click());
 		return new BrickFormulaEditTextDataInteractionWrapper(dataInteraction);
 	}
+
+	public BrickFormulaEditTextDataInteractionWrapper performNewVariable(String variableName)
```

Comment:
since this is kind of special for this kind of test (since everywhere else its sufficient to just use the "new" in spinner) i would prefer to have this as a function in the test class.